src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+  
+    <form 
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <div class="bigSearchBar">
```

Comment:
why do we need this additional wrapper? it's redundant

Reply:
especially since you didn't use this class at all

Reply:
maybe now i don't use it.
but as far as i know, a lot of developers say that it is better to make a div wrapper for each elements

Reply:
can you explain why it is better to use margin top?

Reply:
no, it's better to remove this, you already have outer `form` block